collect_app/src/main/java/org/odk/collect/android/geo/SettingsDialogFragment.java

```diff
@@ -35,7 +36,7 @@
     private int accuracyThresholdIndex = -1;
 
     @Override
-    public void onAttach(Context context) {
+    public void onAttach(@NotNull Context context) {
```

Comment:
How I wish these annotations were more standardized! Since this is an android class, we should use `NonNull` which is already imported here. I think it makes sense to use this one for classes that are not related to android at all.

Reply:
Oh right, fixed.